vulture/core.py

```diff
@@ -218,7 +218,7 @@ def exclude_file(name):
 
         unique_imports = set(item.name for item in self.defined_imports)
         for import_name in unique_imports:
-            path = os.path.join('whitelists', import_name) + '.py'
+            path = os.path.join('whitelists', import_name) + '_whitelist.py'
```

Comment:
The argparse file should be changed as well.

Reply:
Actually, I was wondering if we can directly access `epilog` since it's just an argument to the `ArgumentParser` class. Is it possible without actually instantiating the class?

Reply:
If not, we can just instantiate it.

Reply:
Hey @jendrikseipp,

I just conducted a little experiment:

```
In [1]: from vulture import Vulture

In [2]: v = Vulture()

In [3]: v.scan("""\
   ...: import argparse
   ...:
   ...: parser = argparse.ArgumentParser(epilog=True)
   ...: """)

In [4]: v.get_unused_code()
Out[4]: ['parser']
```

Why do we want to whitelist `epilog` when Vulture isn't reporting as unused? Did I miss something (which is very probable given how sharp I am :D)?

Reply:
The epliog variable here will be reported as unsued: https://bitbucket.org/jendrikseipp/lab/src/d459dc7d09b120d9c156a866f574101216bb4cd4/lab/experiment.py#lines-38